components/breadcrumb/index.jsx

```diff
@@ -70,6 +70,10 @@ Breadcrumb.propTypes = {
 	assistiveText: PropTypes.shape({
 		label: PropTypes.string,
 	}),
+	/**
+	 * Custom styles to be passed to the top-level `nav` tag
+	 */
+	style: PropTypes.object,
```

Comment:
This should be named something else. `styleNav` or `styleContainer`

`style` and `className` should only be used when applied to `.slds-breadcrumb`

Reply:
Done!